test/types/community/changes_from_36.test-d.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import type { PeerCertificate } from 'tls';
+import type { PropExists } from '../utility_types';
+import { expectAssignable, expectError, expectNotType, expectType } from 'tsd';
+import {
+  LoggerLevel,
+  MongoClient,
+  MongoClientOptions,
+  ReadPreference,
+  ReadPreferenceMode
+} from '../../../src';
+
+type MongoDBImport = typeof import('../../../src');
+
+const mongodb: MongoDBImport = (null as unknown) as MongoDBImport;
+
+expectNotType<Function>(mongodb);
+expectType<PropExists<MongoDBImport, 'connect'>>(false);
+// (mongodb.ObjectId);
+
+// MongoClientOptions
+const options: MongoClientOptions = {};
+// .readPreference no longer accepts boolean
+expectNotType<boolean>(options.readPreference);
+// .pkFactory cannot be an empty object
+expectNotType<{}>(options.pkFactory);
+// .checkServerIdentity cannot be `true`
+expectNotType<true>(options.checkServerIdentity);
+// .sslCA cannot be string[]
+expectNotType<string[]>(options.sslCA);
+// .sslCRL cannot be string[]
+expectNotType<string[]>(options.sslCRL);
+// .sslCert cannot be a Buffer
+expectNotType<Buffer>(options.sslCert);
+// .sslKey cannot be a Buffer
+expectNotType<Buffer>(options.sslKey);
+// .sslPass cannot be a Buffer
+expectNotType<Buffer>(options.sslPass);
+
+// Legacy option kept?
```

Comment:
question mark?

Reply:
We dropped all other top level writeConcern options, so I wasn't sure if this was an oversight because it is still present.